modules/ROOT/pages/_partials/include-insights-charts.adoc

```diff
@@ -0,0 +1,385 @@
+// API DETAILS: Requests tab
+//-------------------------
+//tag::api-detail-requests[]
+Request Volume by HTTP Method::
+Total number of requests at specific time points, grouped by the HTTP method used for the request
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+Request Volume by Response Code::
+Total number of requests at specific time points, grouped by the response code (such as *2xx*)
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+// TODO: paths -> Paths (SATHISH)
+// TODO: top 20 paths
+Request Volume by Top N paths::
+Total number of requests at specific time points from the 20 endpoints with the most requests
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+//end::api-detail-requests[]
+//------------------------------------------------------------//
+//
+//-------------------------------------------------------------//
+// API DETAILS: Errors tab
+//-------------------------
+//tag::api-detail-errors[]
+Failed Requests Volume by HTTP Method::
+Total number of unsuccessful requests at specific time points, grouped by the HTTP method used
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+// TODO: downloads available?
+Failed Requests Volume by Response Code::
+Total number of unsuccessful requests at specific time points, grouped by the response code (such as *2xx*)
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+// TODO: paths -> Paths and "N" to "20" if that's the limit (SATHISH)
+Failed Requests by Top N Paths::
+Total number of unsuccessful requests at specific time points from the 20 endpoints with the most requests
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//end::api-detail-errors[]
+//
+//
+//-------------------------------------------------------------//
+// API DETAILS: Performance tab
+//----------------------------
+//tag::api-detail-performance[]
+
+//TODO: issue: RESPONSE TIME UNIT MISSING from chart (SATHISH)
+//
+//can download as CSV, XLS
+Top 20 Paths by Average Response Time::
+Average response time of the 20 most requested endpoints 
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+
+//TODO: 95p (to standardize) or capitalize "percentile" -> "Percentile" (SATHISH)
+//TODO: issue: missing time units on X axis
+//TODO: issue: duplication of time values along the X axis
+//You can download as CSV, XLS
+Top 20 Paths by 95th Percentile Response Time::
+95th percentile value for each of the 20 most requested endpoints
++
+include::partial$include-insights-charts.adoc[tag="download-as"]
+//end::api-detail-performance[]
+//-------------------------------------------------------------//
+//
+//-------------------------------------------------------------//
+// API DETAILS: Client Applications tab
+//-------------------------
+//tag::api-detail-client-apps[]
+Request Volume by Location::
+Maps with the number of requests to geographic areas across the globe
```

Comment:
Do you need "across the globe"?

Reply:
Yes, something like that because it's a complete world map, not an isolated region, such as the UI.